Projects/SEC/SEC04/SEC04-AWS100.md

```diff
@@ -0,0 +1,54 @@
+
+# SEC04-AWS100 — Create an IAM user
+
+## Cloud Service Provider
+
+* Amazon Web Services
+
+## Difficulty
+Level 100 (Introductory)
+
+## Estimated time:
+ 1 hour
+
+## Project's Author(s)
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
+* Enable MFA for the IAM root user 
+* Apply an IAM password policy with best security practises.
```

Comment:
- Typo: practices (practise is a verb)

Reply:
I feel so dumb now!  #100daysofGrammar haha thanks @what-name !